src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let carCost = 0;
+  const costForDays = days * 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const mediumTerm = 3;
+  const mediumTermDiscount = 20;
+
+  if (days >= 1 && days <= 2) {
+    carCost = costForDays;
```

Comment:
You can simplify the function by removing the first condition `if (days >= 1 && days <= 2)`. Since the calculation `costForDays` already accounts for the daily rate, you can return it directly if none of the discount conditions are met. This adheres to checklist item 4 and 5, which suggest using `if` with `return` to simplify conditions and to avoid unnecessary `else` after a `return` statement.